appendix.rst

```diff
@@ -1,70 +0,0 @@
-Appendix: Topics
```

Comment:
Is this file completely gone?
The file itself is probably redundant after this PR, but are the "Recommended readings" preserved somewhere?

Reply:
> are the "Recommended readings" preserved somewhere?

No. Where would you suggest, if we should keep them?

A

Reply:
As @ezio-melotti suggests right below, they could go under the Audience categories (Contributor, Triager, Documentarian, CoreDev) in the restored table in the index. That would be a lot better than reverting whole removal and is the same or less work than reverting the removal since it would mean updating the doc links to ref links and fixing the target names, or fixing the outdated file paths and names, and also adding the pages that were added/split and now missing here or in the wrong sections.

Reply:
> updating the doc links to ref links and fixing the target names, or fixing the outdated file paths and names, and also adding the pages that were added/split and now missing here or in the wrong sections.

I did these in a commit that I kept, so they should still work. Time-wise I probably don't have time to go through and update the table in the next week or so, so reverting was the quickest option.

A

Reply:
For the table (as far as I can tell), but not the appendix, at least as of the version you've pushed here—it still uses `:doc:` links, and almost all of them are now broken, as can be seen [on the preview](https://cpython-devguide--901.org.readthedocs.build/appendix/). Furthermore, it is missing the documents split as part of this PR, leaving it incomplete.

Instead of tediously updating all of that (or introducing a major regression), only to remove it again in a followup PR, I suggest just spending the same or less time re-dleeting it and adding them to the table instead as nessesary. Additionally, to keep it shorter and less work to update, we can just link the top-level index pages of sections that are obviously of central relevance to the audience type in their entirety, instead of linking every sub-page.

 I'm happy to suggest an outline for it, or make a PR to your branch to implement it if you don't have time. Here's what I recommend:

<details>

<summary>Suggested table structure (by column heading)</summary>

All contributors:
* Getting Started
* Following Python's Development
* Issue Tracker
* Running and Writing Tests

Documentarians:
* Documentation
* Development Cycle
* Experts Index
* Triaging an Issue

Triagers:
* Issues and Triaging
* Development Workflow
* Experts Index
* Accepting Pull Requests

Core Developers:
* Core Developers
* Development Workflow
* Issues and Triaging
* Testing and Buildbots

</details>